docs/apis/Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM golang:1.17
+FROM golang:1.21
 
 RUN apt-get update && apt-get -y upgrade && apt-get -y install git
 
 RUN git clone https://github.com/ahmetb/gen-crd-api-reference-docs
+# replace the version pattern in the Dockerfile until we don't move to golang 1.22
+RUN sed -i 's/1.21.0/1.21/g' gen-crd-api-reference-docs/go.mod
```

Comment:
Why do we need this ?

Reply:
Because the Golang 1.22 pattern in the go.mod.
Golang 1.21 does not accept 1.21.0.